reconcile/database_access_manager.py

```diff
@@ -121,8 +130,38 @@ def _generate_db_access(self) -> str:
         ]
         return "\n".join(statements)
 
+    def _generate_revoke_changed(self) -> str:
+        if not self.current_db_access:
+            return ""
+        statements: list[str] = []
+        current_grants = {
+            x.target.dbschema: x.grants for x in self.current_db_access.access or []
+        }
+        desired_grants = {
+            x.target.dbschema: x.grants for x in self.db_access.access or []
+        }
```

Comment:
need to ensure uniq of `target.dbschema` in schema, otherwise this config will cause unexpected drifts.

```yaml
access:
- grants:
  - SELECT
  target:
    dbschema: public
- grants:
  - UPDATE
  target:
    dbschema: public
```

Reply:
Good point, fixed